### ISSUE TITLE
crdroid.net website doesn't work in Russian Federation

### ISSUE THREAD

**Issue by NikitaSavc** [crdroidandroid/issue_tracker#550]
### Issue type
Website issue

###  Description

Seems crdroid.net website protected by Cloudflare.
Cloudflare recently enabled ECH (Encrypted Client Hello) for all users.
This technology allowed users to access blocked resources in Russia, since TSCT (Technical System for Countering Threats) does not see which domain we are accessing.

As a result, Roskomnadzor blocked cloudflare-ech.com
Please disable Cloudflare ECH, so that users from Russia can access the сrdroid.net website

### Steps to reproduce

1. Enter crdroid.net from Russia
2. Cannot open website, we have page "Website blocked by Roskomnadzor"

### Solution

Disable Cloudflare ECH (Encrypted Client Hello)

**Comment by gwolf2u** [comment]
Hello there.
Unfortunately at the time of writing this, there's no way of disabling ECH with the cloudflare plan we use.
Also for any workarounds, would have to lower some specific connection settings, a thing won't do.
So for the time being, there's no way to fix this for our russian users unfortunately.
There are some things you can do from your end, but those are outside the scope of this issue.

**Comment by NikitaSavc** [comment]
@gwolf2u you can disable Cloudflare ECH on any plan (free too) with curl (need global api key and zone ID)

Instruction:

Getting global API key
Enter https://dash.cloudflare.com/profile/api-tokens
And copy your global API key

Getting Zone ID
Enter to settings of your domain, scroll page to down and find "Zone ID". Copy this.

From terminal, enter:
```shell
curl -X PATCH "https://api.cloudflare.com/client/v4/zones/{ID_ZONE}/settings/ech" \
     -H "X-Auth-Email: {ACCOUNT_EMAIL}" \
     -H "X-Auth-Key: {GLOBAL_API_KEY}" \
     -H "Content-Type:application/json" --data '{"id":"ech","value":"off"}'
```

Congratulations, you have disabled Cloudflare ECH.

**Comment by NikitaSavc** [comment]
@gwolf2u check
https://neonode.cc/en/blog/how_to_disable_ech_cf/

**Comment by gwolf2u** [comment]
turned off
feel free to check
